frontend/src/scenes/session-recordings/templates/SessionRecordingTemplates.tsx

```diff
@@ -157,7 +157,9 @@ const RecordingTemplateCard = (props: RecordingTemplateCardProps): JSX.Element =
             closeable={variablesVisible}
             onClose={hideVariables}
             focused={variablesVisible}
-            data-attr={`session-replay-template-${props.category}-${props.template.key}`}
+            data-attr="session-replay-template"
```

Comment:
this makes the data attr concrete so i can have a single autocaptured action for all templates